src/systemctl/systemctl.c

```diff
@@ -534,6 +534,9 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
+        if (invoked_as(argv, "userctl"))
+                arg_runtime_scope = RUNTIME_SCOPE_USER;
```

Comment:
Hmm, maybe additionally refuse the `--system` option when running as userctl?

Reply:
systemctl --user is also accepted, so I prefer to keep the symmetry there